packages/govuk-frontend/src/govuk/components/task-list/template.njk

```diff
@@ -0,0 +1,36 @@
+{% from "../tag/macro.njk" import govukTag -%}
+
+{% set idPrefix = params.idPrefix if params.idPrefix else "task-list" %}
+<ul class="govuk-task-list {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+  {% for item in params.items %}
+    {% if item.hint %}
+      {%- set hintId = idPrefix + "-" + loop.index + "-hint" -%}
+    {% endif %}
+    {%- set statusId = idPrefix + "-" + loop.index + "-status" -%}
+    <li class="govuk-task-list__item {%- if item.href %} govuk-task-list__item--with-link{% endif %}{%- if item.classes %} {{ item.classes }}{% endif %}">
+      <div class="govuk-task-list__task-name-and-hint">
+        {% if item.href %}
+          <a class="govuk-link govuk-task-list__link {%- if item.title.classes %} {{ item.title.classes }}{% endif %}" href="{{ item.href }}" aria-describedby="{{ hintId + " " if item.hint }}{{ statusId }}">{{ item.title.html | safe if item.title.html else item.title.text }}</a>
+        {% else %}
+          <div class="{%- if item.title.classes %} {{ item.title.classes }}{% endif %}">{{ item.title.html | safe if item.title.html else item.title.text }}</div>
```

Comment:
This could result in an empty `class` attribute

```suggestion
          <div{% if item.title.classes %} class="{{ item.title.classes }}" {%- endif %}>{{ item.title.html | safe if item.title.html else item.title.text }}</div>
```